wallet/psbt.go

```diff
@@ -227,6 +232,8 @@ func (w *Wallet) FundPsbt(packet *psbt.Packet, account uint32,
 //
 // NOTE: This method does NOT publish the transaction after it's been finalized
 // successfully.
+//
+// TODO: require account and check if watch only to avoid signing.
```

Comment:
Require account?

Reply:
This was a WIP TODO, it was addressed in a later commit.